src/functions/functionsHelp.component.tsx

```diff
@@ -107,7 +108,12 @@ const FunctionsHelp = (props: FunctionsHelpProps) => {
         . The Wizard will suggest suitable options and indicate using a grey box
         when each item has been recognised.
       </Body>
-      <Heading>Functions and operators included </Heading>
+      <Heading>
+        Functions and operators included{' '}
+        <Typography variant="caption" component="span" fontWeight="normal">
+          (Hover over name for more information)
```

Comment:
In my opinion, `Hover over function name for more information` is more explicit & helpful